components/breadcrumb/src/index.js

```diff
@@ -63,6 +63,7 @@ const BreadcrumbListItem = styled('li')({
   },
 
   // Additions to help ensure links get coloured as we expect
+  // NB specificity of this should override default in Link component
```

Comment:
😢 is this from govuk-frontend?

Reply:
hmmm - i had thought it was, but actually it isn't

on a quick look just now, it seems that breadcrumb doesn't work quite the same.  might need some more looking into to work out what exactly should be going on here

Reply:
having looked a bit further, our breadcrumb component differs slightly from govuk-frontend, i believe to make it "more reacty"...

the underlying issue here is with how the link should get styled.  we had given examples that used `asAnchor`, which have now been changed to use `Link`.  a better solution would be to provide a `Breadcrumb.Link` component/atom that provides styles specific to the breadcrumb system, and for the examples to be changed to use that

Reply:
ok - styling bodge removed - new `Breadcrumb.Link` atom added and docs/examples changed accordingly